sympy/polys/domains/algebraicfield.py

```diff
@@ -26,8 +26,10 @@ def __init__(self, dom, *ext):
             raise DomainError("ground domain must be a rational field")
 
         from sympy.polys.numberfields import to_number_field
-
-        self.orig_ext = ext
+        if len(ext) == 1 and type(ext[0]) is tuple:
```

Comment:
`if len(ext) == 1 and isinstance(ext[0], tuple)`

Reply:
Sure updating it now. 
thanks for pointing it out.